CHANGES.md

```diff
@@ -31,6 +31,12 @@ OpenSSL 3.6
 
 ### Changes between 3.5 and 3.6 [xx XXX xxxx]
 
+ * Changed DirName output format. Previously, DirName was using non-standard single-byte
+   output format `/C=CN/ST...`. The new format is RFC22533 (`C = CN, ST = ...`), and handles
+   multi-byte output correctly.
```

Comment:
The lines are too long and there is a typo in the RFC number.

Reply:
Fixed